includes/wikia/nirvana/WikiaApp.class.php

```diff
@@ -634,7 +634,8 @@ public function sendRequest( $controllerName = null, $methodName = null, $params
 		$request = new WikiaRequest($params);
 
 		if (false == $internal) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1125)